lib/config/base.js

```diff
@@ -11,7 +11,7 @@ module.exports = [
    */
   {
     files: ['**/*.{gts,gjs}'],
-    parser: 'eslint-plugin-ember/gjs-gts-parser',
+    parser: 'ember-eslint-parser',
     processor: 'ember/<noop>',
```

Comment:
@NullVoxPopuli you also have the change this one. And delete the local one in this branch

Reply:
https://github.com/ember-cli/eslint-plugin-ember/blob/d0fa05ec57f0b36931aaa3b1d813750f70c68bbf/lib/index.js#L19

Reply:
https://github.com/NullVoxPopuli/ember-eslint-parser/pull/10

Reply:
there are a few tests that require this preprocessor -- do they still need to?

Reply:
They should use the one from the ember-eslint-parser then